### ISSUE TITLE
How to build/package OMLibrary within debian? 

### ISSUE THREAD

**Issue by Trophime** [OpenModelica/OpenModelicaBuildScripts#4]
Hi,
I'm a newbie..

I would like to rebuild OpenModelica packages for latest Debian/Buster or Ubuntu/Bionic.
To start with, I've tried to reproduce the package for Debian 9.
I works fine...

Now I'm trying to build and package the OMLibrary (remove the --without-omlibrary flag in debian/rules)
It seems that git cannot be performed in pdebuild...

Is there a source package for the omlibrary somewhere?
Thanks for any pointers

Best
C

**Comment by sjoelund** [comment]
There are already Ubuntu/Bionic packages available. The library source packages are available in its repository, or here: https://build.openmodelica.org/apt/pool/libraries/

You can also generate the library packages from: https://github.com/OpenModelica/OMLibraries (`make debian` after configuring some locations). Note that these packages are separate for good reason (my plan is to make different OpenModelica versions possible to install in parallel and share the installed libraries).

**Comment by AnHeuermann** [comment]
Outdated. Please open a new issue if still relevant.
